### DISCUSSION TITLE
API calling the wrong information

### DISCUSSION THREAD

**Discussion by syrup-panini-bit** [transitland/transitland#445] [Q&A] [answered]
I'm trying to call the stop data for LADOT using https://transit.land/api/v2/rest/stops and having the operator_onestop_id be equal to the 3 different ids available: o-9q5-ladot, o-9q5ct-ladot, and o-9q5c-ladot. However, whenever I call it, it outputs information related with "f-u0nkbh2-aziendavaresinatrasporti". Is there a way to fix this?

**Comment by drewda** [comment, marked as answer]
See the documentation for the REST stops endpoint at https://www.transit.land/documentation/rest-api/stops for a list of the allowed query parameters.

To look up by one or more operators that provide service to the route, use `served_by_onestop_ids`

**Comment by dc-reece** [comment]
@mmfJW at 2026-08-01 20:12 UTC — Deploy live. Production smoke successful (`0.13.9` @ `72e3989fac7e1374e7e9e0b87d837ac7563bf794`). Closing out release.
